pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/Connection.java

```diff
@@ -222,4 +232,28 @@ public ResultSetGroup get(long timeout, TimeUnit unit)
       return new ResultSetGroup(response);
     }
   }
+
+
+  public static class Config {
+    public static final String FAIL_ON_EXCEPTIONS = "fail_on_exceptions";
```

Comment:
We don't usually use `_` in the config key, so suggest using camel case for consistency